Leanplum-SDK/Classes/Managers/Networking/LPFileTransferManager.m

```diff
@@ -274,6 +274,8 @@ - (void)downloadFile:(NSString *)path onResponse:(LPNetworkResponseBlock)respons
         id<LPNetworkOperationProtocol> op;
         if ([path hasPrefix:@"http://"] || [path hasPrefix:@"https://"]) {
             op = [self.engine operationWithURLString:path];
+        } else if ([self.filenameToURL valueForKey:path]) {
+            op = [self.engine operationWithURLString:[self.filenameToURL valueForKey:path]];
```

Comment:
should we still check whether the value here has prefix http or https? other than that lgtm

Reply:
that's unnecessary. The filenameToURL should always be giving us a URL